package.json

```diff
@@ -27,6 +27,7 @@
     "handlebars": "4.7.6",
     "jquery-ui-themeroller": "0.0.5",
     "lzma": "2.3.2",
+    "modern-syslog": "^1.2.0",
```

Comment:
Can you skip the `^` to be consistent with other dependencies? If we want to start using ranges, this should be done for all packages at the same time (separately, later).

Reply:
Thank you, I've reverted this for my next commit.